pjsip/include/pjsip/sip_transport.h

```diff
@@ -233,6 +250,8 @@ typedef struct pjsip_tpselector
     /** The type of data in the union */
     pjsip_tpselector_type   type;
 
+    pjsip_tpselector_config config;
+
```

Comment:
- perhaps this can be categorized as criteria, so it can be moved to be inside `union u`?
- also, term `config` is perhaps too broad and currently the criteria is an enum (while `config` is more like a struct that can be expanded in the future), so what if using term like `ip_ver` instead?